.cirrus.yml

```diff
@@ -0,0 +1,13 @@
+freebsd_task:
+  freebsd_instance:
+    matrix:
+    - image: freebsd-13-2-release-amd64
+  build_script:
+    - mkdir build
+    - cd build
+    - ../configure
+    - make -j$(sysctl -n hw.ncpu)
```

Comment:
Please add:

```
- make pythoninfo
```

It's a tool to dump various kind of informations about the platform to ease debug.

Look at the end of: https://api.cirrus-ci.com/v1/task/6753062602145792/logs/build.log

I don't know if it's possible to add a 3rd "pythoninfo_script:" section? Well, it's not needed, I'm fine with adding it at the end of ``build_script``.